docs/tutorial/security.md

```diff
@@ -364,7 +364,12 @@ handler:
 const { session } = require('electron')
 
 session.defaultSession.webRequest.onHeadersReceived((details, callback) => {
-  callback({ responseHeaders: `default-src 'none'` })
+  callback({
+    responseHeaders: {
+      ...details.responseHeaders,
+      "Content-Security-Policy": ["default-src 'none'"]
```

Comment:
Should use single-quotes for consistency.

Reply:
Agreed @masatokinugawa 

Something like:

```suggestion
      'Content-Security-Policy': ['default-src \'none\'']
```